frontend/src/components/App/Notifications/index.tsx

```diff
@@ -93,15 +93,11 @@ function NotificationsList(props: {
           </Grid>
           {!notification.seen && (
             <Grid item md={1}>
-              <Tooltip title={t('translation|Mark as read')}>
-                <IconButton
-                  onClick={e => notificationSeenUnseenHandler(e, notification)}
-                  aria-label={t('translation|Mark as read')}
-                  size="medium"
-                >
-                  <Icon icon="mdi:circle" color={theme.palette.error.main} height={12} width={12} />
```

Comment:
I think the color has not been taken into account in the replacement, right?
We do have notifications not visible for a couple of versions till we fix performance issues, but it would be good to have them look the same.